libraries/AP_BattMonitor/AP_BattMonitor_Params.cpp

```diff
@@ -162,6 +162,13 @@ const AP_Param::GroupInfo AP_BattMonitor_Params::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("ARM_MAH", 19, AP_BattMonitor_Params, _arming_minimum_capacity, 0),
 
+    // @Param: BUS
+    // @DisplayName: I2C bus number
+    // @Description: This specifies the I2C bus number for SMBus based backends
+    // @Values: 0:Bus0(internal),1:Bus1(external),2:Bus2(auxillary),3:Bus3(auxillary2)
+    // @User: Advanced
+    AP_GROUPINFO("BUS", 20, AP_BattMonitor_Params, _bus, 0),
```

Comment:
Not a deal breaker, but ideally this would start going into the subgroup of parameters for a specific backend type so that we could start hiding these from the generic battery backends.

Side note: Do we want to reserve a value that simply tries all the buses till it finds a matching instance to ease the general setup? Past that I'm quite happy for this to be added as it fixes a couple of long outstanding pain points with SMBus batteries.